server.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const SmsProxy = require('./SmsProxy');
+
+const app = express();
+app.set('port', (process.env.PORT || 3000));
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.listen(app.get('port'), function () {
+    console.log('SMS Proxy App listening on port', app.get('port'));
+});
+
+const config = require('dotenv').config();
```

Comment:
In most apps you don't assign `dotenv.config()` to anything. You just call `require('dotenv').config();` as the first line of your app and use `process.env.VARIABLE_NAME` where needed

Reply:
Changed to use `process.env.VARIABLE_NAME`.